pkgs/development/python-modules/vaa/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+
+  #build
```

Comment:
```suggestion
  # build
```

Reply:
I'd be more happy if we didn't have such comments in the first place. We can clearly see in code how these things are used and if they ever move places I'd imagine that they are not updated in the comments. Wrong comments are worse than no comments.

Reply:
Ok